benchmark/buffers/buffer-bytelength-utf8.js

```diff
@@ -0,0 +1,37 @@
+var common = require('../common');
+
+var bench = common.createBenchmark(main, {
+    len: [4, 16, 64, 256, 1024],
+    n: [1e6]
+});
+
+// 16 chars of each byte length
+var chars = {
+  0: 'hello brendan!!!', // 1 byte
+  1: 'ΰαβγδεζηθικλμνξο', // 2 bytes
+  2: '挰挱挲挳挴挵挶挷挸挹挺挻挼挽挾挿', // 3 bytes
+  3: '𠜎𠜱𠝹𠱓𠱸𠲖𠳏𠳕𠴕𠵼𠵿𠸎𠸏𠹷𠺝𠺢' // 4 bytes
+};
+
+function main(conf) {
+  var n = conf.n | 0;
+  var len = conf.len | 0;
+
+  var str = buildStr(len);
+
+  bench.start();
+  for (var i = 0; i < n; i++) {
+    Buffer.byteLength(str, 'utf8');
```

Comment:
This should probably be modified so that `str` is different for every iteration of the loop. Otherwise the JIT will most likely "overly optimize" execution because it's seeing the exact same input every time, which may not give reliable/accurate benchmark results.